README.md

```diff
@@ -1,6 +1,6 @@
 # GOV.UK Publishing Components
 
-[![status](https://badgen.net/github/status/alphagov/govuk_publishing_components/master)](https://ci.integration.publishing.service.gov.uk/job/govuk_publishing_components/job/master/)
+[![status](https://badgen.net/github/status/alphagov/govuk_publishing_components/main)](https://ci.integration.publishing.service.gov.uk/job/govuk_publishing_components/job/main/)
```

Comment:
This repo no longer uses Jenkins for CI – presumably this should now link to:

https://github.com/alphagov/govuk_publishing_components/actions?query=branch%3Amain

or similar?

Reply:
😆 Oh yes, great catch 🤦